GitCommands/Git/RevisionDiffProvider.cs

```diff
@@ -65,7 +109,29 @@ public string Get(string from, string to)
                 to = String.Empty;
             }
 
-            return string.Join(" ", extra, from, to).Trim();
+            if (fileName.IsNotNullOrWhitespace())
```

Comment:
suggest invert the if and exit on the else clause.
should make reading the code and testing easier

Reply:
OK
(smaller diff with this way, easier to review...)